x-pack/plugins/uptime/server/rest_api/synthetics_service/test_now_monitor.ts

```diff
@@ -26,12 +30,21 @@ export const testNowMonitorRoute: UMRestApiRouteFactory = () => ({
       monitorId
     );
 
+    const encryptedClient = server.encryptedSavedObjects.getClient();
+
+    const monitorWithSecrets =
+      await encryptedClient.getDecryptedAsInternalUser<SyntheticsMonitorWithSecrets>(
+        syntheticsMonitor.name,
+        monitorId
+      );
+
     const { syntheticsService } = server;
 
     const testRunId = uuidv4();
 
     const errors = await syntheticsService.triggerConfigs(request, [
       {
+        ...normalizeSecrets(monitorWithSecrets).attributes,
```

Comment:
Should only need one spread. Should be able to get rid of `...monitor.attributes`